web-wallet/src/routes/(welcome)/setup/restore/+page.svelte

```diff
@@ -34,13 +35,19 @@
   let isValidMnemonic = false;
 
   /** @type {string[]} */
-  let mnemonicPhrase = [];
+  let mnemonicPhrase = $mnemonicPhraseResetStore;
 
   const { userId } = $settingsStore;
 
   $: if (showPasswordSetup) {
     password = showPasswordSetup ? password : "";
   }
+
+  onMount(() => {
```

Comment:
Wouldn't be easier to import `onDestroy` instead?

In any case we are missing a test for this behaviour.